lib/iron_bank/resources/product_rate_plan_charge_tier.rb

```diff
@@ -6,6 +6,19 @@ module Resources
     # charge.
     #
     class ProductRatePlanChargeTier < Resource
+      # These fields are declared as `<selectable>true</selectable>` but only
```

Comment:
I'm a little unclear what 'but only ZOQL export.' means here.

Reply:
The key to understand that is in the schema:

```xml
<field>
  <name>Active</name>
  <label>Active</label>
  <selectable>true</selectable>
  <createable>false</createable>
  <updateable>false</updateable>
  <filterable>true</filterable>
  <custom>false</custom>
  <maxlength/>
  <required>false</required>
  <type>boolean</type>
  <contexts>
    <context>export</context>
  </contexts>
</field>
```

^ the above means that this field **is** `selectable` but only in an `export` context. Only field with a `<context>soap</context>` are `selectable` using ZOQL (instead of an export).

Reply:
Got it; I think the wording was just tripping me up a bit.